[Bug] AppenderLoggingException in log4j logging

When importing different kinds of TypeScript projects using the `jssrc2cpg` frontend, CPG creation fails with a `org.apache.logging.log4j.core.appender.AppenderLoggingException`.

**To Reproduce**
Steps to reproduce the behavior:
1. `importCode` the attached projects in joern
[examples.zip](https://github.com/joernio/joern/files/11112219/examples.zip)

2. Observe the following error:
```
2023-03-30 13:25:11,073 main ERROR An exception occurred processing Appender Console org.apache.logging.log4j.core.appender.AppenderLoggingException: java.lang.NoSuchMethodError: 'java.util.Deque org.apache.logging.log4j.util.StackLocatorUtil.getCurrentStackTrace()'
```
<details><summary>Full Stack Trace</summary>
<p>

```java
2023-03-30 13:25:11,073 main ERROR An exception occurred processing Appender Console org.apache.logging.log4j.core.appender.AppenderLoggingException: java.lang.NoSuchMethodError: 'java.util.Deque org.apache.logging.log4j.util.StackLocatorUtil.getCurrentStackTrace()'
	at org.apache.logging.log4j.core.config.AppenderControl.tryCallAppender(AppenderControl.java:165)
	at org.apache.logging.log4j.core.config.AppenderControl.callAppender0(AppenderControl.java:134)
	at org.apache.logging.log4j.core.config.AppenderControl.callAppenderPreventRecursion(AppenderControl.java:125)
	at org.apache.logging.log4j.core.config.AppenderControl.callAppender(AppenderControl.java:89)
	at org.apache.logging.log4j.core.config.LoggerConfig.callAppenders(LoggerConfig.java:683)
	at org.apache.logging.log4j.core.config.LoggerConfig.processLogEvent(LoggerConfig.java:641)
	at org.apache.logging.log4j.core.config.LoggerConfig.log(LoggerConfig.java:624)
	at org.apache.logging.log4j.core.config.LoggerConfig.log(LoggerConfig.java:560)
	at org.apache.logging.log4j.core.config.AwaitCompletionReliabilityStrategy.log(AwaitCompletionReliabilityStrategy.java:82)
	at org.apache.logging.log4j.core.Logger.log(Logger.java:163)
	at org.apache.logging.log4j.spi.AbstractLogger.tryLogMessage(AbstractLogger.java:2205)
	at org.apache.logging.log4j.spi.AbstractLogger.logMessageTrackRecursion(AbstractLogger.java:2159)
	at org.apache.logging.log4j.spi.AbstractLogger.logMessageSafely(AbstractLogger.java:2142)
	at org.apache.logging.log4j.spi.AbstractLogger.logMessage(AbstractLogger.java:2017)
	at org.apache.logging.log4j.spi.AbstractLogger.logIfEnabled(AbstractLogger.java:1983)
	at org.apache.logging.slf4j.Log4jLogger.error(Log4jLogger.java:315)
	at io.shiftleft.passes.ForkJoinParallelCpgPass.createApplySerializeAndStore(CpgPass.scala:84)
	at io.shiftleft.semanticcpg.layers.LayerCreator.runPass(LayerCreator.scala:53)
	at io.joern.dataflowengineoss.layers.dataflows.OssDataFlow.$anonfun$create$1(OssDataFlow.scala:26)
	at io.joern.dataflowengineoss.layers.dataflows.OssDataFlow.$anonfun$create$1$adapted(OssDataFlow.scala:25)
	at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:563)
	at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:561)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1293)
	at io.joern.dataflowengineoss.layers.dataflows.OssDataFlow.create(OssDataFlow.scala:25)
	at io.shiftleft.semanticcpg.layers.LayerCreator.run(LayerCreator.scala:32)
	at io.joern.console.Console.runCreator(Console.scala:434)
	at io.joern.console.Console.$anonfun$_runAnalyzer$1(Console.scala:423)
	at io.joern.console.Console.$anonfun$_runAnalyzer$1$adapted(Console.scala:410)
	at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:563)
	at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:561)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:926)
	at io.joern.console.Console._runAnalyzer(Console.scala:410)
	at io.joern.joerncli.console.JoernConsole.applyDefaultOverlays(JoernConsole.scala:63)
	at io.joern.console.cpgcreation.ImportCode.$anonfun$apply$6(ImportCode.scala:143)
	at scala.Option.map(Option.scala:242)
	at io.joern.console.cpgcreation.ImportCode.io$joern$console$cpgcreation$ImportCode$$apply(ImportCode.scala:140)
	at io.joern.console.cpgcreation.ImportCode.apply(ImportCode.scala:40)
	at ammonite.$sess.cmd2$.<clinit>(cmd2.sc:1)
	at ammonite.$sess.cmd2.$main(cmd2.sc)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at ammonite.runtime.Evaluator$$anon$1.$anonfun$evalMain$1(Evaluator.scala:108)
	at ammonite.util.Util$.withContextClassloader(Util.scala:24)
	at ammonite.runtime.Evaluator$$anon$1.evalMain(Evaluator.scala:90)
	at ammonite.runtime.Evaluator$$anon$1.$anonfun$processLine$2(Evaluator.scala:127)
	at ammonite.util.Catching.map(Res.scala:117)
	at ammonite.runtime.Evaluator$$anon$1.$anonfun$processLine$1(Evaluator.scala:121)
	at ammonite.util.Res$Success.flatMap(Res.scala:62)
	at ammonite.runtime.Evaluator$$anon$1.processLine(Evaluator.scala:120)
	at ammonite.interp.Interpreter.$anonfun$evaluateLine$5(Interpreter.scala:307)
	at ammonite.util.Res$Success.flatMap(Res.scala:62)
	at ammonite.interp.Interpreter.$anonfun$evaluateLine$2(Interpreter.scala:290)
	at ammonite.util.Catching.flatMap(Res.scala:115)
	at ammonite.interp.Interpreter.evaluateLine(Interpreter.scala:289)
	at ammonite.interp.Interpreter.$anonfun$processLine$6(Interpreter.scala:277)
	at ammonite.util.Res$Success.flatMap(Res.scala:62)
	at ammonite.interp.Interpreter.$anonfun$processLine$4(Interpreter.scala:260)
	at ammonite.util.Res$Success.flatMap(Res.scala:62)
	at ammonite.interp.Interpreter.$anonfun$processLine$2(Interpreter.scala:253)
	at ammonite.util.Catching.flatMap(Res.scala:115)
	at ammonite.interp.Interpreter.processLine(Interpreter.scala:252)
	at ammonite.repl.Repl.$anonfun$action$8(Repl.scala:199)
	at ammonite.util.Res$Success.flatMap(Res.scala:62)
	at ammonite.repl.Repl.$anonfun$action$4(Repl.scala:186)
	at ammonite.repl.Scoped.$anonfun$flatMap$1(Signaller.scala:45)
	at ammonite.repl.Signaller.apply(Signaller.scala:28)
	at ammonite.repl.Scoped.flatMap(Signaller.scala:45)
	at ammonite.repl.Scoped.flatMap$(Signaller.scala:45)
	at ammonite.repl.Signaller.flatMap(Signaller.scala:16)
	at ammonite.repl.Repl.$anonfun$action$2(Repl.scala:178)
	at ammonite.util.Catching.flatMap(Res.scala:115)
	at ammonite.repl.Repl.action(Repl.scala:170)
	at ammonite.repl.Repl.loop$1(Repl.scala:212)
	at ammonite.repl.Repl.run(Repl.scala:227)
	at ammonite.Main.$anonfun$run$1(Main.scala:236)
	at scala.Option.getOrElse(Option.scala:201)
	at ammonite.Main.run(Main.scala:224)
	at io.joern.console.ScriptExecution.startInteractiveShell(BridgeBase.scala:213)
	at io.joern.console.ScriptExecution.startInteractiveShell$(BridgeBase.scala:195)
	at io.joern.joerncli.console.AmmoniteBridge$.startInteractiveShell(AmmoniteBridge.scala:5)
	at io.joern.console.BridgeBase.runAmmonite(BridgeBase.scala:163)
	at io.joern.console.BridgeBase.runAmmonite$(BridgeBase.scala:147)
	at io.joern.joerncli.console.AmmoniteBridge$.runAmmonite(AmmoniteBridge.scala:5)
	at io.joern.joerncli.console.AmmoniteBridge$.delayedEndpoint$io$joern$joerncli$console$AmmoniteBridge$1(AmmoniteBridge.scala:7)
	at io.joern.joerncli.console.AmmoniteBridge$delayedInit$body.apply(AmmoniteBridge.scala:5)
	at scala.Function0.apply$mcV$sp(Function0.scala:39)
	at scala.Function0.apply$mcV$sp$(Function0.scala:39)
	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17)
	at scala.App.$anonfun$main$1(App.scala:76)
	at scala.App.$anonfun$main$1$adapted(App.scala:76)
	at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:563)
	at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:561)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:926)
	at scala.App.main(App.scala:76)
	at scala.App.main$(App.scala:74)
	at io.joern.joerncli.console.AmmoniteBridge$.main(AmmoniteBridge.scala:5)
	at io.joern.joerncli.console.AmmoniteBridge.main(AmmoniteBridge.scala)
Caused by: java.lang.NoSuchMethodError: 'java.util.Deque org.apache.logging.log4j.util.StackLocatorUtil.getCurrentStackTrace()'
	at org.apache.logging.log4j.core.impl.ThrowableProxy.<init>(ThrowableProxy.java:112)
	at org.apache.logging.log4j.core.impl.ThrowableProxy.<init>(ThrowableProxy.java:97)
	at org.apache.logging.log4j.core.impl.MutableLogEvent.getThrownProxy(MutableLogEvent.java:354)
	at org.apache.logging.log4j.core.pattern.ExtendedThrowablePatternConverter.format(ExtendedThrowablePatternConverter.java:63)
	at org.apache.logging.log4j.core.layout.PatternLayout$NoFormatPatternSerializer.toSerializable(PatternLayout.java:342)
	at org.apache.logging.log4j.core.layout.PatternLayout.toText(PatternLayout.java:240)
	at org.apache.logging.log4j.core.layout.PatternLayout.encode(PatternLayout.java:225)
	at org.apache.logging.log4j.core.layout.PatternLayout.encode(PatternLayout.java:59)
	at org.apache.logging.log4j.core.appender.AbstractOutputStreamAppender.directEncodeEvent(AbstractOutputStreamAppender.java:215)
	at org.apache.logging.log4j.core.appender.AbstractOutputStreamAppender.tryAppend(AbstractOutputStreamAppender.java:208)
	at org.apache.logging.log4j.core.appender.AbstractOutputStreamAppender.append(AbstractOutputStreamAppender.java:199)
	at org.apache.logging.log4j.core.config.AppenderControl.tryCallAppender(AppenderControl.java:161)
	... 98 more

overflowdb.SchemaViolationException: OUT edge with label REF to an adjacent METHOD is mandatory, but not defined for this METHOD_REF node with id=828
  io.shiftleft.codepropertygraph.generated.nodes.MethodRefDb.referencedMethod(MethodRef.scala:850)
  io.shiftleft.codepropertygraph.generated.nodes.MethodRef.referencedMethod(MethodRef.scala:353)
io.shiftleft.codepropertygraph.generated.traversal.MethodRefTraversalExtGen$.$anonfun$referencedMethod$1(MethodRef.scala:14)
  scala.collection.Iterator$$anon$9.next(Iterator.scala:577)
  scala.collection.Iterator$$anon$10.nextCur(Iterator.scala:587)
  scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:601)
  scala.collection.Iterator$$anon$7.hasNext(Iterator.scala:516)
  scala.collection.Iterator$$anon$6.hasNext(Iterator.scala:470)
  scala.collection.mutable.Growable.addAll(Growable.scala:61)
  scala.collection.mutable.Growable.addAll$(Growable.scala:57)
  scala.collection.mutable.ArrayBuilder.addAll(ArrayBuilder.scala:66)
  scala.collection.IterableOnceOps.toArray(IterableOnce.scala:1282)
  scala.collection.IterableOnceOps.toArray$(IterableOnce.scala:1276)
  scala.collection.AbstractIterator.toArray(Iterator.scala:1293)
  scala.Array$.from(Array.scala:74)
  scala.collection.immutable.ArraySeq$.from(ArraySeq.scala:282)
  scala.collection.immutable.ArraySeq$.from(ArraySeq.scala:273)
  scala.collection.ClassTagIterableFactory$AnyIterableDelegate.from(Factory.scala:671)
  scala.collection.IterableFactory$ToFactory.fromSpecific(Factory.scala:274)
  scala.collection.IterableOnceOps.to(IterableOnce.scala:1246)
  scala.collection.IterableOnceOps.to$(IterableOnce.scala:1246)
  overflowdb.traversal.Traversal.to(Traversal.scala:16)
  overflowdb.traversal.Traversal.sortBy(Traversal.scala:286)
  io.joern.dataflowengineoss.package$.identifiersFromCapturedScopes(package.scala:19)
  io.joern.dataflowengineoss.package$.$anonfun$identifierToFirstUsages$1(package.scala:14)
  scala.collection.Iterator$$anon$10.nextCur(Iterator.scala:587)
  scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:601)
  scala.collection.immutable.List.prependedAll(List.scala:152)
  scala.collection.IterableOnceOps.toList(IterableOnce.scala:1251)
  scala.collection.IterableOnceOps.toList$(IterableOnce.scala:1251)
  scala.collection.AbstractIterator.toList(Iterator.scala:1293)
  overflowdb.traversal.Traversal.l(Traversal.scala:27)
  io.joern.dataflowengineoss.package$.identifierToFirstUsages(package.scala:14)
  io.joern.dataflowengineoss.passes.reachingdef.DdgGenerator.$anonfun$addReachingDefEdges$22(DdgGenerator.scala:175)
  scala.collection.Iterator$$anon$10.nextCur(Iterator.scala:587)
  scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:601)
  scala.collection.immutable.List.prependedAll(List.scala:152)
  scala.collection.IterableOnceOps.toList(IterableOnce.scala:1251)
  scala.collection.IterableOnceOps.toList$(IterableOnce.scala:1251)
  scala.collection.AbstractIterator.toList(Iterator.scala:1293)
  overflowdb.traversal.Traversal.l(Traversal.scala:27)
io.joern.dataflowengineoss.passes.reachingdef.DdgGenerator.addEdgesToCapturedIdentifiersAndParameters$1(DdgGenerator.scala:174)
  io.joern.dataflowengineoss.passes.reachingdef.DdgGenerator.addReachingDefEdges(DdgGenerator.scala:208)
  io.joern.dataflowengineoss.passes.reachingdef.ReachingDefPass.runOnPart(ReachingDefPass.scala:31)
  io.joern.dataflowengineoss.passes.reachingdef.ReachingDefPass.runOnPart(ReachingDefPass.scala:14)
  io.shiftleft.passes.NewStyleCpgPassBase$$anon$2.accept(CpgPass.scala:147)
  io.shiftleft.passes.NewStyleCpgPassBase$$anon$2.accept(CpgPass.scala:145)
  java.util.stream.ReduceOps$4ReducingSink.accept(ReduceOps.java:220)
  java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:948)
  java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
  java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
  java.util.stream.ReduceOps$ReduceTask.doLeaf(ReduceOps.java:952)
  java.util.stream.ReduceOps$ReduceTask.doLeaf(ReduceOps.java:926)
  java.util.stream.AbstractTask.compute(AbstractTask.java:327)
  java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:746)
  java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
  java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
  java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
  java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
  java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
java.util.NoSuchElementException: next on empty iterator
  scala.collection.Iterator$$anon$19.next(Iterator.scala:966)
  scala.collection.Iterator$$anon$19.next(Iterator.scala:964)
  scala.collection.Iterator$$anon$7.next(Iterator.scala:528)
  overflowdb.traversal.Traversal.next(Traversal.scala:22)
  io.shiftleft.codepropertygraph.generated.nodes.MethodRefDb.referencedMethod(MethodRef.scala:845)
  io.shiftleft.codepropertygraph.generated.nodes.MethodRef.referencedMethod(MethodRef.scala:353)
io.shiftleft.codepropertygraph.generated.traversal.MethodRefTraversalExtGen$.$anonfun$referencedMethod$1(MethodRef.scala:14)
  scala.collection.Iterator$$anon$9.next(Iterator.scala:577)
  scala.collection.Iterator$$anon$10.nextCur(Iterator.scala:587)
  scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:601)
  scala.collection.Iterator$$anon$7.hasNext(Iterator.scala:516)
  scala.collection.Iterator$$anon$6.hasNext(Iterator.scala:470)
  scala.collection.mutable.Growable.addAll(Growable.scala:61)
  scala.collection.mutable.Growable.addAll$(Growable.scala:57)
  scala.collection.mutable.ArrayBuilder.addAll(ArrayBuilder.scala:66)
  scala.collection.IterableOnceOps.toArray(IterableOnce.scala:1282)
  scala.collection.IterableOnceOps.toArray$(IterableOnce.scala:1276)
  scala.collection.AbstractIterator.toArray(Iterator.scala:1293)
  scala.Array$.from(Array.scala:74)
  scala.collection.immutable.ArraySeq$.from(ArraySeq.scala:282)
  scala.collection.immutable.ArraySeq$.from(ArraySeq.scala:273)
  scala.collection.ClassTagIterableFactory$AnyIterableDelegate.from(Factory.scala:671)
  scala.collection.IterableFactory$ToFactory.fromSpecific(Factory.scala:274)
  scala.collection.IterableOnceOps.to(IterableOnce.scala:1246)
  scala.collection.IterableOnceOps.to$(IterableOnce.scala:1246)
  overflowdb.traversal.Traversal.to(Traversal.scala:16)
  overflowdb.traversal.Traversal.sortBy(Traversal.scala:286)
  io.joern.dataflowengineoss.package$.identifiersFromCapturedScopes(package.scala:19)
  io.joern.dataflowengineoss.package$.$anonfun$identifierToFirstUsages$1(package.scala:14)
  scala.collection.Iterator$$anon$10.nextCur(Iterator.scala:587)
  scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:601)
  scala.collection.immutable.List.prependedAll(List.scala:152)
  scala.collection.IterableOnceOps.toList(IterableOnce.scala:1251)
  scala.collection.IterableOnceOps.toList$(IterableOnce.scala:1251)
  scala.collection.AbstractIterator.toList(Iterator.scala:1293)
  overflowdb.traversal.Traversal.l(Traversal.scala:27)
  io.joern.dataflowengineoss.package$.identifierToFirstUsages(package.scala:14)
  io.joern.dataflowengineoss.passes.reachingdef.DdgGenerator.$anonfun$addReachingDefEdges$22(DdgGenerator.scala:175)
  scala.collection.Iterator$$anon$10.nextCur(Iterator.scala:587)
  scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:601)
  scala.collection.immutable.List.prependedAll(List.scala:152)
  scala.collection.IterableOnceOps.toList(IterableOnce.scala:1251)
  scala.collection.IterableOnceOps.toList$(IterableOnce.scala:1251)
  scala.collection.AbstractIterator.toList(Iterator.scala:1293)
  overflowdb.traversal.Traversal.l(Traversal.scala:27)
io.joern.dataflowengineoss.passes.reachingdef.DdgGenerator.addEdgesToCapturedIdentifiersAndParameters$1(DdgGenerator.scala:174)
  io.joern.dataflowengineoss.passes.reachingdef.DdgGenerator.addReachingDefEdges(DdgGenerator.scala:208)
  io.joern.dataflowengineoss.passes.reachingdef.ReachingDefPass.runOnPart(ReachingDefPass.scala:31)
  io.joern.dataflowengineoss.passes.reachingdef.ReachingDefPass.runOnPart(ReachingDefPass.scala:14)
  io.shiftleft.passes.NewStyleCpgPassBase$$anon$2.accept(CpgPass.scala:147)
  io.shiftleft.passes.NewStyleCpgPassBase$$anon$2.accept(CpgPass.scala:145)
  java.util.stream.ReduceOps$4ReducingSink.accept(ReduceOps.java:220)
  java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:948)
  java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
  java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
  java.util.stream.ReduceOps$ReduceTask.doLeaf(ReduceOps.java:952)
  java.util.stream.ReduceOps$ReduceTask.doLeaf(ReduceOps.java:926)
  java.util.stream.AbstractTask.compute(AbstractTask.java:327)
  java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:746)
  java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
  java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
  java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
  java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
  java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
```

</p>
</details> 

**Expected behavior**
A CPG should be created.

**Desktop (please complete the following information):**
 - OS: Linux 5.15.0-1030-gcp 37-Ubuntu
 - Joern Version: 1.1.1552 
 - javac 11.0.18

**Additional context**
As can be seen in the full stack trace above, the bug mentioned in the introduction is not the only one being thrown but there is also an `overflowdb.SchemaViolationException`.

I am not certain, if `AppenderLoggingException` is the root cause or a manifestation but I thought the bug could be due to recent changes for the upgrade to Java 19.

Can not reproduce this SchemaViolationException at all.

But we end up with:

```
[ERROR] Pass io.joern.jssrc2cpg.passes.JavaScriptTypeRecovery failed
java.util.concurrent.ExecutionException: overflowdb.traversal.filter.StringPropertyFilter$InvalidRegexException: invalid regular expression: `.*{startOffset, endOffset, replacement}.*`
        at java.util.concurrent.ForkJoinTask.get(ForkJoinTask.java:1006) ~[?:?]
        at io.joern.x2cpg.passes.frontend.XTypeRecovery.$anonfun$run$3(XTypeRecovery.scala:92) ~[io.joern.x2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.x2cpg.passes.frontend.XTypeRecovery.$anonfun$run$3$adapted(XTypeRecovery.scala:92) ~[io.joern.x2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:563) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:561) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at overflowdb.traversal.Traversal.foreach(Traversal.scala:16) ~[io.shiftleft.overflowdb-traversal_2.13-1.169.jar:1.169]
        at io.joern.x2cpg.passes.frontend.XTypeRecovery.run(XTypeRecovery.scala:92) ~[io.joern.x2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.shiftleft.passes.CpgPass.runOnPart(CpgPass.scala:27) ~[io.shiftleft.codepropertygraph_2.13-1.3.597.jar:1.3.597]
        at io.shiftleft.passes.NewStyleCpgPassBase.runWithBuilder(CpgPass.scala:134) ~[io.shiftleft.codepropertygraph_2.13-1.3.597.jar:1.3.597]
        at io.shiftleft.passes.ForkJoinParallelCpgPass.createApplySerializeAndStore(CpgPass.scala:74) ~[io.shiftleft.codepropertygraph_2.13-1.3.597.jar:1.3.597]
        at io.shiftleft.passes.NewStyleCpgPassBase.createAndApply(CpgPass.scala:124) ~[io.shiftleft.codepropertygraph_2.13-1.3.597.jar:1.3.597]
        at io.joern.x2cpg.passes.frontend.XTypeRecoveryPass.$anonfun$run$1(XTypeRecovery.scala:59) ~[io.joern.x2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at scala.collection.immutable.Range.foreach$mVc$sp(Range.scala:190) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at io.joern.x2cpg.passes.frontend.XTypeRecoveryPass.run(XTypeRecovery.scala:58) ~[io.joern.x2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.shiftleft.passes.CpgPass.runOnPart(CpgPass.scala:27) ~[io.shiftleft.codepropertygraph_2.13-1.3.597.jar:1.3.597]
        at io.shiftleft.passes.NewStyleCpgPassBase.runWithBuilder(CpgPass.scala:134) ~[io.shiftleft.codepropertygraph_2.13-1.3.597.jar:1.3.597]
        at io.shiftleft.passes.ForkJoinParallelCpgPass.createApplySerializeAndStore(CpgPass.scala:74) ~[io.shiftleft.codepropertygraph_2.13-1.3.597.jar:1.3.597]
        at io.shiftleft.passes.NewStyleCpgPassBase.createAndApply(CpgPass.scala:124) ~[io.shiftleft.codepropertygraph_2.13-1.3.597.jar:1.3.597]
        at io.joern.console.cpgcreation.JsSrcCpgGenerator.$anonfun$applyPostProcessingPasses$1(JsSrcCpgGenerator.scala:24) ~[io.joern.console-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.console.cpgcreation.JsSrcCpgGenerator.$anonfun$applyPostProcessingPasses$1$adapted(JsSrcCpgGenerator.scala:24) ~[io.joern.console-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at scala.collection.immutable.List.foreach(List.scala:333) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at io.joern.console.cpgcreation.JsSrcCpgGenerator.applyPostProcessingPasses(JsSrcCpgGenerator.scala:24) ~[io.joern.console-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.console.cpgcreation.ImportCode.$anonfun$apply$6(ImportCode.scala:144) ~[io.joern.console-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at scala.Option.map(Option.scala:242) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at io.joern.console.cpgcreation.ImportCode.io$joern$console$cpgcreation$ImportCode$$apply(ImportCode.scala:140) ~[io.joern.console-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.console.cpgcreation.ImportCode.apply(ImportCode.scala:40) ~[io.joern.console-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at ammonite.$sess.cmd0$.<clinit>(cmd0.sc:1) ~[?:?]
        at ammonite.$sess.cmd0.$main(cmd0.sc) ~[?:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
        at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
        at ammonite.runtime.Evaluator$$anon$1.$anonfun$evalMain$1(Evaluator.scala:108) ~[com.lihaoyi.ammonite-runtime_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.util.Util$.withContextClassloader(Util.scala:24) ~[com.lihaoyi.ammonite-util_2.13-2.5.8.jar:2.5.8]
        at ammonite.runtime.Evaluator$$anon$1.evalMain(Evaluator.scala:90) ~[com.lihaoyi.ammonite-runtime_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.runtime.Evaluator$$anon$1.$anonfun$processLine$2(Evaluator.scala:127) ~[com.lihaoyi.ammonite-runtime_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.util.Catching.map(Res.scala:117) ~[com.lihaoyi.ammonite-util_2.13-2.5.8.jar:2.5.8]
        at ammonite.runtime.Evaluator$$anon$1.$anonfun$processLine$1(Evaluator.scala:121) ~[com.lihaoyi.ammonite-runtime_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.util.Res$Success.flatMap(Res.scala:62) ~[com.lihaoyi.ammonite-util_2.13-2.5.8.jar:2.5.8]
        at ammonite.runtime.Evaluator$$anon$1.processLine(Evaluator.scala:120) ~[com.lihaoyi.ammonite-runtime_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.interp.Interpreter.$anonfun$evaluateLine$5(Interpreter.scala:307) ~[com.lihaoyi.ammonite-interp_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.util.Res$Success.flatMap(Res.scala:62) ~[com.lihaoyi.ammonite-util_2.13-2.5.8.jar:2.5.8]
        at ammonite.interp.Interpreter.$anonfun$evaluateLine$2(Interpreter.scala:290) ~[com.lihaoyi.ammonite-interp_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.util.Catching.flatMap(Res.scala:115) ~[com.lihaoyi.ammonite-util_2.13-2.5.8.jar:2.5.8]
        at ammonite.interp.Interpreter.evaluateLine(Interpreter.scala:289) ~[com.lihaoyi.ammonite-interp_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.interp.Interpreter.$anonfun$processLine$6(Interpreter.scala:277) ~[com.lihaoyi.ammonite-interp_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.util.Res$Success.flatMap(Res.scala:62) ~[com.lihaoyi.ammonite-util_2.13-2.5.8.jar:2.5.8]
        at ammonite.interp.Interpreter.$anonfun$processLine$4(Interpreter.scala:260) ~[com.lihaoyi.ammonite-interp_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.util.Res$Success.flatMap(Res.scala:62) ~[com.lihaoyi.ammonite-util_2.13-2.5.8.jar:2.5.8]
        at ammonite.interp.Interpreter.$anonfun$processLine$2(Interpreter.scala:253) ~[com.lihaoyi.ammonite-interp_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.util.Catching.flatMap(Res.scala:115) ~[com.lihaoyi.ammonite-util_2.13-2.5.8.jar:2.5.8]
        at ammonite.interp.Interpreter.processLine(Interpreter.scala:252) ~[com.lihaoyi.ammonite-interp_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.repl.Repl.$anonfun$action$8(Repl.scala:199) ~[com.lihaoyi.ammonite-repl_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.util.Res$Success.flatMap(Res.scala:62) ~[com.lihaoyi.ammonite-util_2.13-2.5.8.jar:2.5.8]
        at ammonite.repl.Repl.$anonfun$action$4(Repl.scala:186) ~[com.lihaoyi.ammonite-repl_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.repl.Scoped.$anonfun$flatMap$1(Signaller.scala:45) ~[com.lihaoyi.ammonite-repl_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.repl.Signaller.apply(Signaller.scala:28) ~[com.lihaoyi.ammonite-repl_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.repl.Scoped.flatMap(Signaller.scala:45) ~[com.lihaoyi.ammonite-repl_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.repl.Scoped.flatMap$(Signaller.scala:45) ~[com.lihaoyi.ammonite-repl_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.repl.Signaller.flatMap(Signaller.scala:16) ~[com.lihaoyi.ammonite-repl_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.repl.Repl.$anonfun$action$2(Repl.scala:178) ~[com.lihaoyi.ammonite-repl_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.util.Catching.flatMap(Res.scala:115) ~[com.lihaoyi.ammonite-util_2.13-2.5.8.jar:2.5.8]
        at ammonite.repl.Repl.action(Repl.scala:170) ~[com.lihaoyi.ammonite-repl_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.repl.Repl.loop$1(Repl.scala:212) ~[com.lihaoyi.ammonite-repl_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.repl.Repl.run(Repl.scala:227) ~[com.lihaoyi.ammonite-repl_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.Main.$anonfun$run$1(Main.scala:236) ~[com.lihaoyi.ammonite_2.13.8-2.5.8.jar:2.5.8]
        at scala.Option.getOrElse(Option.scala:201) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at ammonite.Main.run(Main.scala:224) ~[com.lihaoyi.ammonite_2.13.8-2.5.8.jar:2.5.8]
        at io.joern.console.ScriptExecution.startInteractiveShell(BridgeBase.scala:213) ~[io.joern.console-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.console.ScriptExecution.startInteractiveShell$(BridgeBase.scala:195) ~[io.joern.console-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.joerncli.console.AmmoniteBridge$.startInteractiveShell(AmmoniteBridge.scala:5) ~[io.joern.joern-cli-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.console.BridgeBase.runAmmonite(BridgeBase.scala:163) ~[io.joern.console-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.console.BridgeBase.runAmmonite$(BridgeBase.scala:147) ~[io.joern.console-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.joerncli.console.AmmoniteBridge$.runAmmonite(AmmoniteBridge.scala:5) ~[io.joern.joern-cli-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.joerncli.console.AmmoniteBridge$.delayedEndpoint$io$joern$joerncli$console$AmmoniteBridge$1(AmmoniteBridge.scala:7) ~[io.joern.joern-cli-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.joerncli.console.AmmoniteBridge$delayedInit$body.apply(AmmoniteBridge.scala:5) ~[io.joern.joern-cli-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at scala.Function0.apply$mcV$sp(Function0.scala:39) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.Function0.apply$mcV$sp$(Function0.scala:39) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.App.$anonfun$main$1(App.scala:76) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.App.$anonfun$main$1$adapted(App.scala:76) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:563) [org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:561) [org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.collection.AbstractIterable.foreach(Iterable.scala:926) [org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.App.main(App.scala:76) [org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.App.main$(App.scala:74) [org.scala-lang.scala-library-2.13.8.jar:?]
        at io.joern.joerncli.console.AmmoniteBridge$.main(AmmoniteBridge.scala:5) [io.joern.joern-cli-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.joerncli.console.AmmoniteBridge.main(AmmoniteBridge.scala) [io.joern.joern-cli-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
Caused by: overflowdb.traversal.filter.StringPropertyFilter$InvalidRegexException: invalid regular expression: `.*{startOffset, endOffset, replacement}.*`
        at overflowdb.traversal.filter.StringPropertyFilter$.regexpCompile(StringPropertyFilter.scala:51) ~[io.shiftleft.overflowdb-traversal_2.13-1.169.jar:1.169]
        at overflowdb.traversal.filter.StringPropertyFilter$.regexp(StringPropertyFilter.scala:13) ~[io.shiftleft.overflowdb-traversal_2.13-1.169.jar:1.169]
        at io.shiftleft.codepropertygraph.generated.traversal.ExpressionTraversalExtGen$.code$extension(Expression.scala:122) ~[io.shiftleft.codepropertygraph-domain-classes_2.13-1.3.597.jar:1.3.597]
        at io.joern.jssrc2cpg.passes.RecoverForJavaScriptFile.$anonfun$isField$2(JavaScriptTypeRecovery.scala:145) ~[io.joern.jssrc2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at overflowdb.traversal.Traversal.$anonfun$where$1(Traversal.scala:115) ~[io.shiftleft.overflowdb-traversal_2.13-1.169.jar:1.169]
        at overflowdb.traversal.Traversal.$anonfun$where$1$adapted(Traversal.scala:114) ~[io.shiftleft.overflowdb-traversal_2.13-1.169.jar:1.169]
        at scala.collection.Iterator$$anon$6.hasNext(Iterator.scala:472) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.collection.IterableOnceOps.isEmpty(IterableOnce.scala:793) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.collection.IterableOnceOps.isEmpty$(IterableOnce.scala:793) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at overflowdb.traversal.Traversal.isEmpty(Traversal.scala:16) ~[io.shiftleft.overflowdb-traversal_2.13-1.169.jar:1.169]
        at scala.collection.IterableOnceOps.nonEmpty(IterableOnce.scala:800) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.collection.IterableOnceOps.nonEmpty$(IterableOnce.scala:800) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at overflowdb.traversal.Traversal.nonEmpty(Traversal.scala:16) ~[io.shiftleft.overflowdb-traversal_2.13-1.169.jar:1.169]
        at io.joern.jssrc2cpg.passes.RecoverForJavaScriptFile.$anonfun$isField$1(JavaScriptTypeRecovery.scala:145) ~[io.joern.jssrc2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at scala.runtime.java8.JFunction0$mcZ$sp.apply(JFunction0$mcZ$sp.scala:17) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.collection.concurrent.TrieMap.getOrElseUpdate(TrieMap.scala:961) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at io.joern.jssrc2cpg.passes.RecoverForJavaScriptFile.isField(JavaScriptTypeRecovery.scala:146) ~[io.joern.jssrc2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.x2cpg.passes.frontend.RecoverForXCompilationUnit.persistType(XTypeRecovery.scala:904) ~[io.joern.x2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.x2cpg.passes.frontend.RecoverForXCompilationUnit.setTypeInformationForRecCall(XTypeRecovery.scala:850) ~[io.joern.x2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.x2cpg.passes.frontend.RecoverForXCompilationUnit.$anonfun$setTypeInformation$1(XTypeRecovery.scala:782) ~[io.joern.x2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:563) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:561) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at overflowdb.traversal.Traversal.foreach(Traversal.scala:16) ~[io.shiftleft.overflowdb-traversal_2.13-1.169.jar:1.169]
        at io.joern.x2cpg.passes.frontend.RecoverForXCompilationUnit.setTypeInformation(XTypeRecovery.scala:775) ~[io.joern.x2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.x2cpg.passes.frontend.RecoverForXCompilationUnit.compute(XTypeRecovery.scala:255) ~[io.joern.x2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.x2cpg.passes.frontend.RecoverForXCompilationUnit.compute(XTypeRecovery.scala:141) ~[io.joern.x2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at java.util.concurrent.RecursiveTask.exec(RecursiveTask.java:94) ~[?:?]
        at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290) ~[?:?]
        at java.util.concurrent.ForkJoinTask.tryExternalHelp(ForkJoinTask.java:381) ~[?:?]
        at java.util.concurrent.ForkJoinTask.externalInterruptibleAwaitDone(ForkJoinTask.java:351) ~[?:?]
        at java.util.concurrent.ForkJoinTask.get(ForkJoinTask.java:1004) ~[?:?]
        ... 87 more
Caused by: java.util.regex.PatternSyntaxException: Illegal repetition near index 5
(?s).*{startOffset, endOffset, replacement}.*
     ^
        at java.util.regex.Pattern.error(Pattern.java:2027) ~[?:?]
        at java.util.regex.Pattern.closure(Pattern.java:3320) ~[?:?]
        at java.util.regex.Pattern.sequence(Pattern.java:2213) ~[?:?]
        at java.util.regex.Pattern.expr(Pattern.java:2068) ~[?:?]
        at java.util.regex.Pattern.compile(Pattern.java:1782) ~[?:?]
        at java.util.regex.Pattern.<init>(Pattern.java:1428) ~[?:?]
        at java.util.regex.Pattern.compile(Pattern.java:1068) ~[?:?]
        at scala.util.matching.Regex.<init>(Regex.scala:234) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.collection.StringOps$.r$extension(StringOps.scala:857) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at overflowdb.traversal.filter.StringPropertyFilter$.regexpCompile(StringPropertyFilter.scala:48) ~[io.shiftleft.overflowdb-traversal_2.13-1.169.jar:1.169]
        at overflowdb.traversal.filter.StringPropertyFilter$.regexp(StringPropertyFilter.scala:13) ~[io.shiftleft.overflowdb-traversal_2.13-1.169.jar:1.169]
        at io.shiftleft.codepropertygraph.generated.traversal.ExpressionTraversalExtGen$.code$extension(Expression.scala:122) ~[io.shiftleft.codepropertygraph-domain-classes_2.13-1.3.597.jar:1.3.597]
        at io.joern.jssrc2cpg.passes.RecoverForJavaScriptFile.$anonfun$isField$2(JavaScriptTypeRecovery.scala:145) ~[io.joern.jssrc2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at overflowdb.traversal.Traversal.$anonfun$where$1(Traversal.scala:115) ~[io.shiftleft.overflowdb-traversal_2.13-1.169.jar:1.169]
        at overflowdb.traversal.Traversal.$anonfun$where$1$adapted(Traversal.scala:114) ~[io.shiftleft.overflowdb-traversal_2.13-1.169.jar:1.169]
        at scala.collection.Iterator$$anon$6.hasNext(Iterator.scala:472) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.collection.IterableOnceOps.isEmpty(IterableOnce.scala:793) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.collection.IterableOnceOps.isEmpty$(IterableOnce.scala:793) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at overflowdb.traversal.Traversal.isEmpty(Traversal.scala:16) ~[io.shiftleft.overflowdb-traversal_2.13-1.169.jar:1.169]
        at scala.collection.IterableOnceOps.nonEmpty(IterableOnce.scala:800) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.collection.IterableOnceOps.nonEmpty$(IterableOnce.scala:800) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at overflowdb.traversal.Traversal.nonEmpty(Traversal.scala:16) ~[io.shiftleft.overflowdb-traversal_2.13-1.169.jar:1.169]
        at io.joern.jssrc2cpg.passes.RecoverForJavaScriptFile.$anonfun$isField$1(JavaScriptTypeRecovery.scala:145) ~[io.joern.jssrc2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at scala.runtime.java8.JFunction0$mcZ$sp.apply(JFunction0$mcZ$sp.scala:17) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.collection.concurrent.TrieMap.getOrElseUpdate(TrieMap.scala:961) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at io.joern.jssrc2cpg.passes.RecoverForJavaScriptFile.isField(JavaScriptTypeRecovery.scala:146) ~[io.joern.jssrc2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.x2cpg.passes.frontend.RecoverForXCompilationUnit.persistType(XTypeRecovery.scala:904) ~[io.joern.x2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.x2cpg.passes.frontend.RecoverForXCompilationUnit.setTypeInformationForRecCall(XTypeRecovery.scala:850) ~[io.joern.x2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.x2cpg.passes.frontend.RecoverForXCompilationUnit.$anonfun$setTypeInformation$1(XTypeRecovery.scala:782) ~[io.joern.x2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:563) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:561) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at overflowdb.traversal.Traversal.foreach(Traversal.scala:16) ~[io.shiftleft.overflowdb-traversal_2.13-1.169.jar:1.169]
        at io.joern.x2cpg.passes.frontend.RecoverForXCompilationUnit.setTypeInformation(XTypeRecovery.scala:775) ~[io.joern.x2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.x2cpg.passes.frontend.RecoverForXCompilationUnit.compute(XTypeRecovery.scala:255) ~[io.joern.x2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.x2cpg.passes.frontend.RecoverForXCompilationUnit.compute(XTypeRecovery.scala:141) ~[io.joern.x2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at java.util.concurrent.RecursiveTask.exec(RecursiveTask.java:94) ~[?:?]
        at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290) ~[?:?]
        at java.util.concurrent.ForkJoinTask.tryExternalHelp(ForkJoinTask.java:381) ~[?:?]
        at java.util.concurrent.ForkJoinTask.externalInterruptibleAwaitDone(ForkJoinTask.java:351) ~[?:?]
        at java.util.concurrent.ForkJoinTask.get(ForkJoinTask.java:1004) ~[?:?]
        ... 87 more
[INFO ] Pass io.joern.jssrc2cpg.passes.JavaScriptTypeRecovery completed in 462 ms (0% on mutations). -1 + 0 changes committed from 0 parts.
[ERROR] Pass io.joern.jssrc2cpg.passes.JavaScriptTypeRecoveryPass failed
java.util.concurrent.ExecutionException: overflowdb.traversal.filter.StringPropertyFilter$InvalidRegexException: invalid regular expression: `.*{startOffset, endOffset, replacement}.*`
        at java.util.concurrent.ForkJoinTask.get(ForkJoinTask.java:1006) ~[?:?]
        at io.joern.x2cpg.passes.frontend.XTypeRecovery.$anonfun$run$3(XTypeRecovery.scala:92) ~[io.joern.x2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.x2cpg.passes.frontend.XTypeRecovery.$anonfun$run$3$adapted(XTypeRecovery.scala:92) ~[io.joern.x2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:563) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:561) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at overflowdb.traversal.Traversal.foreach(Traversal.scala:16) ~[io.shiftleft.overflowdb-traversal_2.13-1.169.jar:1.169]
        at io.joern.x2cpg.passes.frontend.XTypeRecovery.run(XTypeRecovery.scala:92) ~[io.joern.x2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.shiftleft.passes.CpgPass.runOnPart(CpgPass.scala:27) ~[io.shiftleft.codepropertygraph_2.13-1.3.597.jar:1.3.597]
        at io.shiftleft.passes.NewStyleCpgPassBase.runWithBuilder(CpgPass.scala:134) ~[io.shiftleft.codepropertygraph_2.13-1.3.597.jar:1.3.597]
        at io.shiftleft.passes.ForkJoinParallelCpgPass.createApplySerializeAndStore(CpgPass.scala:74) ~[io.shiftleft.codepropertygraph_2.13-1.3.597.jar:1.3.597]
        at io.shiftleft.passes.NewStyleCpgPassBase.createAndApply(CpgPass.scala:124) ~[io.shiftleft.codepropertygraph_2.13-1.3.597.jar:1.3.597]
        at io.joern.x2cpg.passes.frontend.XTypeRecoveryPass.$anonfun$run$1(XTypeRecovery.scala:59) ~[io.joern.x2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at scala.collection.immutable.Range.foreach$mVc$sp(Range.scala:190) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at io.joern.x2cpg.passes.frontend.XTypeRecoveryPass.run(XTypeRecovery.scala:58) ~[io.joern.x2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.shiftleft.passes.CpgPass.runOnPart(CpgPass.scala:27) ~[io.shiftleft.codepropertygraph_2.13-1.3.597.jar:1.3.597]
        at io.shiftleft.passes.NewStyleCpgPassBase.runWithBuilder(CpgPass.scala:134) ~[io.shiftleft.codepropertygraph_2.13-1.3.597.jar:1.3.597]
        at io.shiftleft.passes.ForkJoinParallelCpgPass.createApplySerializeAndStore(CpgPass.scala:74) ~[io.shiftleft.codepropertygraph_2.13-1.3.597.jar:1.3.597]
        at io.shiftleft.passes.NewStyleCpgPassBase.createAndApply(CpgPass.scala:124) ~[io.shiftleft.codepropertygraph_2.13-1.3.597.jar:1.3.597]
        at io.joern.console.cpgcreation.JsSrcCpgGenerator.$anonfun$applyPostProcessingPasses$1(JsSrcCpgGenerator.scala:24) ~[io.joern.console-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.console.cpgcreation.JsSrcCpgGenerator.$anonfun$applyPostProcessingPasses$1$adapted(JsSrcCpgGenerator.scala:24) ~[io.joern.console-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at scala.collection.immutable.List.foreach(List.scala:333) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at io.joern.console.cpgcreation.JsSrcCpgGenerator.applyPostProcessingPasses(JsSrcCpgGenerator.scala:24) ~[io.joern.console-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.console.cpgcreation.ImportCode.$anonfun$apply$6(ImportCode.scala:144) ~[io.joern.console-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at scala.Option.map(Option.scala:242) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at io.joern.console.cpgcreation.ImportCode.io$joern$console$cpgcreation$ImportCode$$apply(ImportCode.scala:140) ~[io.joern.console-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.console.cpgcreation.ImportCode.apply(ImportCode.scala:40) ~[io.joern.console-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at ammonite.$sess.cmd0$.<clinit>(cmd0.sc:1) ~[?:?]
        at ammonite.$sess.cmd0.$main(cmd0.sc) ~[?:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
        at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
        at ammonite.runtime.Evaluator$$anon$1.$anonfun$evalMain$1(Evaluator.scala:108) ~[com.lihaoyi.ammonite-runtime_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.util.Util$.withContextClassloader(Util.scala:24) ~[com.lihaoyi.ammonite-util_2.13-2.5.8.jar:2.5.8]
        at ammonite.runtime.Evaluator$$anon$1.evalMain(Evaluator.scala:90) ~[com.lihaoyi.ammonite-runtime_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.runtime.Evaluator$$anon$1.$anonfun$processLine$2(Evaluator.scala:127) ~[com.lihaoyi.ammonite-runtime_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.util.Catching.map(Res.scala:117) ~[com.lihaoyi.ammonite-util_2.13-2.5.8.jar:2.5.8]
        at ammonite.runtime.Evaluator$$anon$1.$anonfun$processLine$1(Evaluator.scala:121) ~[com.lihaoyi.ammonite-runtime_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.util.Res$Success.flatMap(Res.scala:62) ~[com.lihaoyi.ammonite-util_2.13-2.5.8.jar:2.5.8]
        at ammonite.runtime.Evaluator$$anon$1.processLine(Evaluator.scala:120) ~[com.lihaoyi.ammonite-runtime_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.interp.Interpreter.$anonfun$evaluateLine$5(Interpreter.scala:307) ~[com.lihaoyi.ammonite-interp_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.util.Res$Success.flatMap(Res.scala:62) ~[com.lihaoyi.ammonite-util_2.13-2.5.8.jar:2.5.8]
        at ammonite.interp.Interpreter.$anonfun$evaluateLine$2(Interpreter.scala:290) ~[com.lihaoyi.ammonite-interp_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.util.Catching.flatMap(Res.scala:115) ~[com.lihaoyi.ammonite-util_2.13-2.5.8.jar:2.5.8]
        at ammonite.interp.Interpreter.evaluateLine(Interpreter.scala:289) ~[com.lihaoyi.ammonite-interp_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.interp.Interpreter.$anonfun$processLine$6(Interpreter.scala:277) ~[com.lihaoyi.ammonite-interp_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.util.Res$Success.flatMap(Res.scala:62) ~[com.lihaoyi.ammonite-util_2.13-2.5.8.jar:2.5.8]
        at ammonite.interp.Interpreter.$anonfun$processLine$4(Interpreter.scala:260) ~[com.lihaoyi.ammonite-interp_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.util.Res$Success.flatMap(Res.scala:62) ~[com.lihaoyi.ammonite-util_2.13-2.5.8.jar:2.5.8]
        at ammonite.interp.Interpreter.$anonfun$processLine$2(Interpreter.scala:253) ~[com.lihaoyi.ammonite-interp_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.util.Catching.flatMap(Res.scala:115) ~[com.lihaoyi.ammonite-util_2.13-2.5.8.jar:2.5.8]
        at ammonite.interp.Interpreter.processLine(Interpreter.scala:252) ~[com.lihaoyi.ammonite-interp_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.repl.Repl.$anonfun$action$8(Repl.scala:199) ~[com.lihaoyi.ammonite-repl_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.util.Res$Success.flatMap(Res.scala:62) ~[com.lihaoyi.ammonite-util_2.13-2.5.8.jar:2.5.8]
        at ammonite.repl.Repl.$anonfun$action$4(Repl.scala:186) ~[com.lihaoyi.ammonite-repl_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.repl.Scoped.$anonfun$flatMap$1(Signaller.scala:45) ~[com.lihaoyi.ammonite-repl_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.repl.Signaller.apply(Signaller.scala:28) ~[com.lihaoyi.ammonite-repl_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.repl.Scoped.flatMap(Signaller.scala:45) ~[com.lihaoyi.ammonite-repl_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.repl.Scoped.flatMap$(Signaller.scala:45) ~[com.lihaoyi.ammonite-repl_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.repl.Signaller.flatMap(Signaller.scala:16) ~[com.lihaoyi.ammonite-repl_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.repl.Repl.$anonfun$action$2(Repl.scala:178) ~[com.lihaoyi.ammonite-repl_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.util.Catching.flatMap(Res.scala:115) ~[com.lihaoyi.ammonite-util_2.13-2.5.8.jar:2.5.8]
        at ammonite.repl.Repl.action(Repl.scala:170) ~[com.lihaoyi.ammonite-repl_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.repl.Repl.loop$1(Repl.scala:212) ~[com.lihaoyi.ammonite-repl_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.repl.Repl.run(Repl.scala:227) ~[com.lihaoyi.ammonite-repl_2.13.8-2.5.8.jar:2.5.8]
        at ammonite.Main.$anonfun$run$1(Main.scala:236) ~[com.lihaoyi.ammonite_2.13.8-2.5.8.jar:2.5.8]
        at scala.Option.getOrElse(Option.scala:201) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at ammonite.Main.run(Main.scala:224) ~[com.lihaoyi.ammonite_2.13.8-2.5.8.jar:2.5.8]
        at io.joern.console.ScriptExecution.startInteractiveShell(BridgeBase.scala:213) ~[io.joern.console-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.console.ScriptExecution.startInteractiveShell$(BridgeBase.scala:195) ~[io.joern.console-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.joerncli.console.AmmoniteBridge$.startInteractiveShell(AmmoniteBridge.scala:5) ~[io.joern.joern-cli-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.console.BridgeBase.runAmmonite(BridgeBase.scala:163) ~[io.joern.console-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.console.BridgeBase.runAmmonite$(BridgeBase.scala:147) ~[io.joern.console-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.joerncli.console.AmmoniteBridge$.runAmmonite(AmmoniteBridge.scala:5) ~[io.joern.joern-cli-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.joerncli.console.AmmoniteBridge$.delayedEndpoint$io$joern$joerncli$console$AmmoniteBridge$1(AmmoniteBridge.scala:7) ~[io.joern.joern-cli-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.joerncli.console.AmmoniteBridge$delayedInit$body.apply(AmmoniteBridge.scala:5) ~[io.joern.joern-cli-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at scala.Function0.apply$mcV$sp(Function0.scala:39) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.Function0.apply$mcV$sp$(Function0.scala:39) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.App.$anonfun$main$1(App.scala:76) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.App.$anonfun$main$1$adapted(App.scala:76) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:563) [org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:561) [org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.collection.AbstractIterable.foreach(Iterable.scala:926) [org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.App.main(App.scala:76) [org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.App.main$(App.scala:74) [org.scala-lang.scala-library-2.13.8.jar:?]
        at io.joern.joerncli.console.AmmoniteBridge$.main(AmmoniteBridge.scala:5) [io.joern.joern-cli-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.joerncli.console.AmmoniteBridge.main(AmmoniteBridge.scala) [io.joern.joern-cli-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
Caused by: overflowdb.traversal.filter.StringPropertyFilter$InvalidRegexException: invalid regular expression: `.*{startOffset, endOffset, replacement}.*`
        at overflowdb.traversal.filter.StringPropertyFilter$.regexpCompile(StringPropertyFilter.scala:51) ~[io.shiftleft.overflowdb-traversal_2.13-1.169.jar:1.169]
        at overflowdb.traversal.filter.StringPropertyFilter$.regexp(StringPropertyFilter.scala:13) ~[io.shiftleft.overflowdb-traversal_2.13-1.169.jar:1.169]
        at io.shiftleft.codepropertygraph.generated.traversal.ExpressionTraversalExtGen$.code$extension(Expression.scala:122) ~[io.shiftleft.codepropertygraph-domain-classes_2.13-1.3.597.jar:1.3.597]
        at io.joern.jssrc2cpg.passes.RecoverForJavaScriptFile.$anonfun$isField$2(JavaScriptTypeRecovery.scala:145) ~[io.joern.jssrc2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at overflowdb.traversal.Traversal.$anonfun$where$1(Traversal.scala:115) ~[io.shiftleft.overflowdb-traversal_2.13-1.169.jar:1.169]
        at overflowdb.traversal.Traversal.$anonfun$where$1$adapted(Traversal.scala:114) ~[io.shiftleft.overflowdb-traversal_2.13-1.169.jar:1.169]
        at scala.collection.Iterator$$anon$6.hasNext(Iterator.scala:472) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.collection.IterableOnceOps.isEmpty(IterableOnce.scala:793) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.collection.IterableOnceOps.isEmpty$(IterableOnce.scala:793) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at overflowdb.traversal.Traversal.isEmpty(Traversal.scala:16) ~[io.shiftleft.overflowdb-traversal_2.13-1.169.jar:1.169]
        at scala.collection.IterableOnceOps.nonEmpty(IterableOnce.scala:800) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.collection.IterableOnceOps.nonEmpty$(IterableOnce.scala:800) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at overflowdb.traversal.Traversal.nonEmpty(Traversal.scala:16) ~[io.shiftleft.overflowdb-traversal_2.13-1.169.jar:1.169]
        at io.joern.jssrc2cpg.passes.RecoverForJavaScriptFile.$anonfun$isField$1(JavaScriptTypeRecovery.scala:145) ~[io.joern.jssrc2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at scala.runtime.java8.JFunction0$mcZ$sp.apply(JFunction0$mcZ$sp.scala:17) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.collection.concurrent.TrieMap.getOrElseUpdate(TrieMap.scala:961) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at io.joern.jssrc2cpg.passes.RecoverForJavaScriptFile.isField(JavaScriptTypeRecovery.scala:146) ~[io.joern.jssrc2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.x2cpg.passes.frontend.RecoverForXCompilationUnit.persistType(XTypeRecovery.scala:904) ~[io.joern.x2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.x2cpg.passes.frontend.RecoverForXCompilationUnit.setTypeInformationForRecCall(XTypeRecovery.scala:850) ~[io.joern.x2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.x2cpg.passes.frontend.RecoverForXCompilationUnit.$anonfun$setTypeInformation$1(XTypeRecovery.scala:782) ~[io.joern.x2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:563) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:561) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at overflowdb.traversal.Traversal.foreach(Traversal.scala:16) ~[io.shiftleft.overflowdb-traversal_2.13-1.169.jar:1.169]
        at io.joern.x2cpg.passes.frontend.RecoverForXCompilationUnit.setTypeInformation(XTypeRecovery.scala:775) ~[io.joern.x2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.x2cpg.passes.frontend.RecoverForXCompilationUnit.compute(XTypeRecovery.scala:255) ~[io.joern.x2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.x2cpg.passes.frontend.RecoverForXCompilationUnit.compute(XTypeRecovery.scala:141) ~[io.joern.x2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at java.util.concurrent.RecursiveTask.exec(RecursiveTask.java:94) ~[?:?]
        at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290) ~[?:?]
        at java.util.concurrent.ForkJoinTask.tryExternalHelp(ForkJoinTask.java:381) ~[?:?]
        at java.util.concurrent.ForkJoinTask.externalInterruptibleAwaitDone(ForkJoinTask.java:351) ~[?:?]
        at java.util.concurrent.ForkJoinTask.get(ForkJoinTask.java:1004) ~[?:?]
        ... 87 more
Caused by: java.util.regex.PatternSyntaxException: Illegal repetition near index 5
(?s).*{startOffset, endOffset, replacement}.*
     ^
        at java.util.regex.Pattern.error(Pattern.java:2027) ~[?:?]
        at java.util.regex.Pattern.closure(Pattern.java:3320) ~[?:?]
        at java.util.regex.Pattern.sequence(Pattern.java:2213) ~[?:?]
        at java.util.regex.Pattern.expr(Pattern.java:2068) ~[?:?]
        at java.util.regex.Pattern.compile(Pattern.java:1782) ~[?:?]
        at java.util.regex.Pattern.<init>(Pattern.java:1428) ~[?:?]
        at java.util.regex.Pattern.compile(Pattern.java:1068) ~[?:?]
        at scala.util.matching.Regex.<init>(Regex.scala:234) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.collection.StringOps$.r$extension(StringOps.scala:857) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at overflowdb.traversal.filter.StringPropertyFilter$.regexpCompile(StringPropertyFilter.scala:48) ~[io.shiftleft.overflowdb-traversal_2.13-1.169.jar:1.169]
        at overflowdb.traversal.filter.StringPropertyFilter$.regexp(StringPropertyFilter.scala:13) ~[io.shiftleft.overflowdb-traversal_2.13-1.169.jar:1.169]
        at io.shiftleft.codepropertygraph.generated.traversal.ExpressionTraversalExtGen$.code$extension(Expression.scala:122) ~[io.shiftleft.codepropertygraph-domain-classes_2.13-1.3.597.jar:1.3.597]
        at io.joern.jssrc2cpg.passes.RecoverForJavaScriptFile.$anonfun$isField$2(JavaScriptTypeRecovery.scala:145) ~[io.joern.jssrc2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at overflowdb.traversal.Traversal.$anonfun$where$1(Traversal.scala:115) ~[io.shiftleft.overflowdb-traversal_2.13-1.169.jar:1.169]
        at overflowdb.traversal.Traversal.$anonfun$where$1$adapted(Traversal.scala:114) ~[io.shiftleft.overflowdb-traversal_2.13-1.169.jar:1.169]
        at scala.collection.Iterator$$anon$6.hasNext(Iterator.scala:472) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.collection.IterableOnceOps.isEmpty(IterableOnce.scala:793) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.collection.IterableOnceOps.isEmpty$(IterableOnce.scala:793) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at overflowdb.traversal.Traversal.isEmpty(Traversal.scala:16) ~[io.shiftleft.overflowdb-traversal_2.13-1.169.jar:1.169]
        at scala.collection.IterableOnceOps.nonEmpty(IterableOnce.scala:800) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.collection.IterableOnceOps.nonEmpty$(IterableOnce.scala:800) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at overflowdb.traversal.Traversal.nonEmpty(Traversal.scala:16) ~[io.shiftleft.overflowdb-traversal_2.13-1.169.jar:1.169]
        at io.joern.jssrc2cpg.passes.RecoverForJavaScriptFile.$anonfun$isField$1(JavaScriptTypeRecovery.scala:145) ~[io.joern.jssrc2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at scala.runtime.java8.JFunction0$mcZ$sp.apply(JFunction0$mcZ$sp.scala:17) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.collection.concurrent.TrieMap.getOrElseUpdate(TrieMap.scala:961) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at io.joern.jssrc2cpg.passes.RecoverForJavaScriptFile.isField(JavaScriptTypeRecovery.scala:146) ~[io.joern.jssrc2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.x2cpg.passes.frontend.RecoverForXCompilationUnit.persistType(XTypeRecovery.scala:904) ~[io.joern.x2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.x2cpg.passes.frontend.RecoverForXCompilationUnit.setTypeInformationForRecCall(XTypeRecovery.scala:850) ~[io.joern.x2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.x2cpg.passes.frontend.RecoverForXCompilationUnit.$anonfun$setTypeInformation$1(XTypeRecovery.scala:782) ~[io.joern.x2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:563) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:561) ~[org.scala-lang.scala-library-2.13.8.jar:?]
        at overflowdb.traversal.Traversal.foreach(Traversal.scala:16) ~[io.shiftleft.overflowdb-traversal_2.13-1.169.jar:1.169]
        at io.joern.x2cpg.passes.frontend.RecoverForXCompilationUnit.setTypeInformation(XTypeRecovery.scala:775) ~[io.joern.x2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.x2cpg.passes.frontend.RecoverForXCompilationUnit.compute(XTypeRecovery.scala:255) ~[io.joern.x2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at io.joern.x2cpg.passes.frontend.RecoverForXCompilationUnit.compute(XTypeRecovery.scala:141) ~[io.joern.x2cpg-1.1.1559+1-b3826a7f.jar:1.1.1559+1-b3826a7f]
        at java.util.concurrent.RecursiveTask.exec(RecursiveTask.java:94) ~[?:?]
        at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290) ~[?:?]
        at java.util.concurrent.ForkJoinTask.tryExternalHelp(ForkJoinTask.java:381) ~[?:?]
        at java.util.concurrent.ForkJoinTask.externalInterruptibleAwaitDone(ForkJoinTask.java:351) ~[?:?]
        at java.util.concurrent.ForkJoinTask.get(ForkJoinTask.java:1004) ~[?:?]
        ... 87 more

```

Looks like another instance of that regex problem you fixed this morning @DavidBakerEffendi

--- a/joern-cli/frontends/jssrc2cpg/src/main/scala/io/joern/jssrc2cpg/passes/JavaScriptTypeRecovery.scala
+++ b/joern-cli/frontends/jssrc2cpg/src/main/scala/io/joern/jssrc2cpg/passes/JavaScriptTypeRecovery.scala
@@ -142,8 +142,7 @@ private class RecoverForJavaScriptFile(cpg: Cpg, cu: File, builder: DiffGraphBui
         .ast
         .assignment
         .code("exports.*")
-        .where(_.argument.code(s".*${i.name}.*"))
-        .nonEmpty || super.isField(i)
+        .exists(_.argument.code.exists(_.contains(i.name))) || super.isField(i)
     )
 
   override protected def visitIdentifierAssignedToConstructor(i: Identifier, c: Call): Set[String] = {

